financial-connections/src/main/java/com/stripe/android/financialconnections/ui/FinancialConnectionsSheetNativeActivity.kt

```diff
@@ -146,43 +144,37 @@ internal class FinancialConnectionsSheetNativeActivity : AppCompatActivity(), Ma
                 viewModel.onBackClick(navController.currentDestination?.pane)
                 if (navController.popBackStack().not()) viewModel.onBackPressed()
             }
-            FinancialConnectionsModalBottomSheetLayout(
-                sheetState = bottomState,
-                sheetContent = {
-                    exitModal?.let {
-                        ExitModal(
-                            loading = it.loading,
-                            description = it.description,
-                            onCancel = { coroutineScope.launch { bottomState.hide() } },
-                            onExit = viewModel::onCloseConfirm
-                        )
-                    }
-                },
-                content = {
-                    NavHost(
-                        navController,
-                        startDestination = initialDestination.fullRoute,
-                    ) {
-                        composable(Destination.Consent)
-                        composable(Destination.ManualEntry)
-                        composable(Destination.PartnerAuth)
-                        composable(Destination.InstitutionPicker)
-                        composable(Destination.AccountPicker)
-                        composable(Destination.Success)
-                        composable(Destination.Reset)
-                        composable(Destination.Error)
-                        composable(Destination.AttachLinkedPaymentAccount)
-                        composable(Destination.NetworkingLinkSignup)
-                        composable(Destination.NetworkingLinkLoginWarmup)
-                        composable(Destination.NetworkingLinkVerification)
-                        composable(Destination.NetworkingSaveToLinkVerification)
-                        composable(Destination.LinkAccountPicker)
-                        composable(Destination.BankAuthRepair)
-                        composable(Destination.LinkStepUpVerification)
-                        composable(Destination.ManualEntrySuccess)
-                    }
+            ModalBottomSheetLayout(
+                bottomSheetNavigator = bottomSheetNavigator,
+                sheetBackgroundColor = FinancialConnectionsTheme.v3Colors.backgroundSurface,
+                sheetShape = RoundedCornerShape(8.dp),
+                scrimColor = Neutral900.copy(alpha = 0.32f),
```

Comment:
We should think about how to incorporate alpha values (and in general: enabled and disabled states) into a Sail design system. Not for this PR, but let’s talk 😊

Reply:
+1!